solution/ui/regulations/eregs-vite/src/components/resources/ResourcesResults.vue

```diff
@@ -107,18 +102,27 @@ export default {
     }
 }
 
-.related-sections {
-    margin-bottom: 40px;
-    font-size: $font-size-xs;
-    color: $mid_gray;
+.resources-content-container {
+    .related-sections {
```

Comment:
add specificity to these Single File Component styles so that they override the Related Sections stylesheet for this specific component.